indico/modules/events/registration/models/forms.py

```diff
@@ -239,6 +247,12 @@ class RegistrationForm(db.Model):
             lazy=True
         )
     )
+    #: The ticket template used in PDF
+    ticket_template = db.relationship(
+        'DesignerTemplate',
+        lazy=True,
+        foreign_keys=ticket_template_id
```

Comment:
Usually we add a backref to almost all relationships. In this case I would call it e.g. `ticket_for_regforms`. Having such relationships - even if not used in the code - makes it easier to do stuff in indico shell.

Don't forget to run `python bin/maintenance/update_backrefs.py` after adding a backref to update the comment in the referenced model.